app/lib/brexit_checker/actions.yaml

```diff
@@ -1222,7 +1405,9 @@ actions:
   guidance_link_text: Exporting objects of cultural interest
   guidance_url: https://www.gov.uk/guidance/exporting-or-importing-objects-of-cultural-interest
   criteria:
-  - culture
+  - all_of:
```

Comment:
There are no changes to T100 on the card

Reply:
Mark's checked, and added to the card 👍